Documentation/ceph-upgrade.md

```diff
@@ -294,26 +286,24 @@ This cluster is finished:
   rook-version=v1.2.0
 ```
 
-### 3. Verify the updated cluster
+### 4. Verify the updated cluster
 
 At this point, your Rook operator should be running version `rook/ceph:v1.2.0`.
 
 Verify the Ceph cluster's health using the [health verification section](#health-verification).
 
-### 4. Update Rook-Ceph custom resource definitions
+### 5. Update Rook-Ceph custom resource definitions
 
 > **IMPORTANT**: Do not perform this step until ALL existing Rook-Ceph clusters are updated!
 
 After all Rook-Ceph clusters have been updated following the steps above, update the Rook-Ceph
-Custom Resource Definitions. This is important to get the latest schema validations that will
-support running external Ceph clusters and will also help with creating or modifying Rook-Ceph
-deployments in the future.
+Custom Resource Definitions. This will help with creating or modifying Rook-Ceph
+deployments in the future with the updated schema validation.
 
 ```sh
 kubectl apply -f upgrade-from-v1.1-crds.yaml
 ```
 
-
```

Comment:
Why removing this line? FYI, I have double-lined splits between L2-headed sections. There is another before the CSI updates section.

Reply:
I don't believe the rendering will show extra lines and @galexrt has also suggested to remove extra lines between sections for some reformatting he's still working on, but I don't have a strong opinion.

Reply:
I don't know that I care too much, but we should be consistent and do the same thing here and the CSI section.. I usually do it to help with seeing section breaks while editing rather than anything for rendered formatting.